native/packages/react-native-bpk-component-icon/stories.js

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
```

Comment:
I tested that Flow works here by changing the `small` prop on an icon to a string (should be a boolean) and it does display an error.